types/xml2js/index.d.ts

```diff
@@ -90,42 +92,8 @@ export interface BuilderOptions {
     cdata?: boolean;
 }
 
-export interface Options {
-    async?: boolean;
-    attrkey?: string;
-    attrNameProcessors?: Array<(name: string) => any>;
-    attrValueProcessors?: Array<(value: string, name: string) => any>;
-    charkey?: string;
-    charsAsChildren?: boolean;
-    childkey?: string;
-    emptyTag?: any;
-    explicitArray?: boolean;
-    explicitCharkey?: boolean;
-    explicitChildren?: boolean;
-    explicitRoot?: boolean;
-    ignoreAttrs?: boolean;
-    includeWhiteChars?: boolean;
-    mergeAttrs?: boolean;
-    normalize?: boolean;
-    normalizeTags?: boolean;
-    strict?: boolean;
-    tagNameProcessors?: Array<(name: string) => any>;
-    trim?: boolean;
-    validator?: Function;
-    valueProcessors?: Array<(value: string, name: string) => any>;
-    xmlns?: boolean;
-}
-
-export interface OptionsV2 extends Options {
-    preserveChildrenOrder?: boolean;
-    rootName?: string;
-    xmldec?: XmlDeclarationAttributes;
-    doctype?: any;
-    renderOpts?: RenderOptions;
-    headless?: boolean;
-    chunkSize?: number;
-    cdata?: boolean;
-}
+export type Options = Omit<ParserOptions, "preserveChildrenOrder" | "chunkSize">;
+export type OptionsV2 = ParserOptions & BuilderOptions;
```

Comment:
It seems this default omits `allowSurrogateChars`

Reply:
@claasahl any thoughts on ^? It's no biggie.

Reply:
The default value for allowSurrogateChars is supposed to be false (or at least falsy) - https://www.npmjs.com/package/xml2js#options-for-the-builder-class